server/cluster/cluster.go

```diff
@@ -849,6 +862,54 @@ func (c *RaftCluster) PutStore(store *metapb.Store) error {
 	return c.putStoreLocked(s)
 }
 
+// PutStoreForce puts a store forcely .
+func (c *RaftCluster) PutStoreForce(store *metapb.Store) error {
```

Comment:
Better to add a flag parameter to `PutStore` rather than add a new function to reduce duplicated code.

Reply:
OK, I will try.

Reply:
Hi, @Sullivan12138  Could you unify `PutStore` and `PutStoreForce` into one function.

Reply:
Ok, I am done.